Sources/BraintreePaymentFlow/WebAuthenticationSession.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import AuthenticationServices
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+class WebAuthenticationSession: NSObject {
```

Comment:
This is a punt for sure, but I still think it makes sense to move this class into Core to prevent needing to make updates across several modules if we want to make changes to our `WebAuthenticationSession` since they all use the exact same logic. We could potentially be in a scenario where we make an update here but forget to in SEPA and have different behavior across modules that's unintended. Nothing to worry about for this particular PR though. 😄

Reply:
I did think about that doing SEPA but I think we don't have the two separate completion handler as we do in PPCP.

Reply:
Here is the ticket I made for it. https://paypal.atlassian.net/browse/DTBTSDK-2688